### PR TITLE
Moved-constants-to-config-for-startinepoch

### PR DESCRIPTION
**Pull request by sasurobert**
moved minNumPeersToConsider and minNumOfPeersToConsiderBlockValid to config.

### CODE REVIEW

File: epochStart/bootstrap/export_test.go [hunk 1]
```diff
@@ -268,5 +268,3 @@ func (e *epochStartMetaBlockProcessor) GetMapMetaBlock() map[string]*block.MetaB
 const DurationBetweenChecksForEpochStartMetaBlock = durationBetweenChecks
 
 const DurationBetweenReRequest = durationBetweenReRequests
-
-const MinNumOfPeersToConsiderBlockValid = minNumOfPeersToConsiderBlockValid
```
Comment (SebastianMarian) [inline epochStart/bootstrap/export_test.go:272]: You can remove also the two consts above

File: epochStart/errors.go [hunk 2]
```diff
@@ -211,3 +211,9 @@ var ErrNilLatestStorageDataProvider = errors.New("nil latest storage data provid
 
 // ErrNilShuffler signals that a nil shuffler was provided
 var ErrNilShuffler = errors.New("nil nodes shuffler provided")
+
+// ErrNotEnoughMinNumConnectedPeers signals that config is invalid for num of connected peers
+var ErrNotEnoughMinNumConnectedPeers = errors.New("not enough min num of connected peers from config")
```
Comment (SebastianMarian) [inline epochStart/errors.go:216]: ErrNotEnoughNumConnectedPeers and  "not enough num connected peers from config"

File: epochStart/bootstrap/export_test.go (hunk 1 above)
Comment (sasurobert) [reply epochStart/bootstrap/export_test.go:272]: done

File: epochStart/errors.go (hunk 2 above)
Comment (sasurobert) [reply epochStart/errors.go:216]: done

File: cmd/node/config/config.toml [hunk 3]
```diff
@@ -413,6 +413,10 @@
     EnableTriggerFromP2P = true
     PublicKeyToListenFrom = "b5ea539180627040b2703cabde8ebece0ded59a0c9d2d4a8a73a299fb7bc11d36ac3d1a620501c230059cffa983a7419d192a50ca55ed8aa07254658dda27aba1e54f50fd7cfbb9e66ce1715d72f9c1b0b05272388f9825742161a211debff15"
 
+[StartInEpochConfig]
+    MinNumConnectedPeersToStart       = 2
+	MinNumOfPeersToConsiderBlockValid = 2
```
Comment (iulianpascalau) [inline cmd/node/config/config.toml:418]: There is an indentation fail in github. Maybe use only white spaces?

**Comment by elrondscammer2** [comment]
Crooked sons of bitches. Give us our money back.

**Comment by elrondscammer2** [comment]
That's enough!! Cheating bastards.
The price keeps falling. People don't trust you. ERD doesn't buy. What do you boys do for a living?

File: cmd/node/config/config.toml (hunk 3 above)
Comment (elrondscammer4) [reply cmd/node/config/config.toml:418]: We will reveal all the facts on our Elrond Scam twitter account soon. Please don't buy ERD. I don't want you to get hurt.

**Pull request merged**